modules/tinymce/src/core/main/ts/fmt/ApplyFormat.ts

```diff
@@ -282,6 +291,7 @@ const applyFormat = (ed: Editor, name: string, vars?: FormatVars, node?: Node |
           node = mergeStyles(node);
         }
 
+        // TODO: Make sure all of the below handle noneditable elements correctly
```

Comment:
Just adding a task for this

- [ ] Make sure this TODO is done and cleaned up

Reply:
TODO has been looked into and handled now